src/Codeception/Module/ZF2.php

```diff
@@ -107,8 +120,28 @@ public function _after(TestCase $test)
     }
 
     public function _getEntityManager()
-    {
-        $serviceLocator = Application::init($this->applicationConfig)->getServiceManager();
-        return $serviceLocator->get('Doctrine\ORM\EntityManager');
+    {   
+        return $this->grabServiceFromContainer('Doctrine\ORM\EntityManager');
```

Comment:
@InVeX yes, i have nothing to this method. I have concerns about `entityManager` property (see above). Please remove that property.